src/runtime/server/server.ts

```diff
@@ -38,7 +39,7 @@ export interface Server {
   }
   express: Express
   handlers: {
-    playground: NexusRequestHandler
+    //  playground: NexusRequestHandler
```

Comment:
OT

💡  `$ nexus playground` with flags, host playground on the fly. Use-case is working with hosted or nextjs. 

counter argument: nextjs users would prefer to continue using a lambda to host their playground. One terminal, no other proess needed, during local development. Yet we say not to do this already in current nextjs integration guide.